doc/README.hive

```diff
@@ -0,0 +1,173 @@
+   information on Hive specific exported environment variables that
+   may be useful in scripts.
+
+5) Hive requires HDFS, so ensure the Hadoop w/ HDFS is configured and
+   also in your submission script.  MapReduce is not needed with Hive
+   but can be setup along with it.  See README.hadoop for Hadoop setup
+   instructions.
+
+6) Hive requires Zookeeper, so setup the essentials for Zookeeper.
+   See README.zookeeper for Zookeeper setup instructions.
+
+7) Submit your job into the cluster by running "sbatch -k
+   ./magpie.sbatchfile" for Slurm, "msub ./magpie.msubfile" for
+   Moab, or "bsub < ./magpie.lsffile" for LSF.
+   Add any other options you see fit.
+
+8) Look at your job output file to see your output.  There will also
+   be some notes/instructions/tips in the output file for viewing the
+   status of your job in a web browser, environment variables you wish
+   to set if interacting with it, etc.
+
+   See "General Advanced Usage" in README for additional tips.
+
+Hive Notes
+-----------
+
+Hive is configured to run on the master node, so increasing or decreasing
+the number of nodes should not have an affect on Hive directly (it may
+have an affect on performance, but Hive will always run on the master).
+
+Hive Exported Environment Variables
+------------------------------------
+
+The following environment variables are exported when your job is run
+and may be useful in scripts in your run or in pre/post run scripts.
+
+HIVE_MASTER_NODE : The node where HiveSever and Hive Metastore are running
+
+HIVE_HOME : Location to the Hive binaries
+
+HIVE_PORT : The port that HiveServer is listening for connections on
+
+HIVE_CONF_DIR : the directory that Hive configuration files local
+                 to the node are stored.
+
+POSTGRES_HOME : Where PostgreSQL binary lives
+
+PGPORT : The port that PostgreSQL will listen for connections on. 11115 works well
+
+PGDATA : The path for PostgreSQL to use for its tables
+
+
+See "Hadoop Exported Environment Variables" in README.hadoop, for
+Hadoop environment variables that may be useful.
+
+See "Zookeeper Exported Environment Variables" in README.zookeeper,
+for Zookeeper environment variables that may be useful.
+
+Hive Patching
+--------------
+
+Typically there are no patches required for Hive to operate.  The only
+contingency is that PostgreSQL is available on the system as well.
+
+For additional functionality/testing purposes you may wish to download the
+hive-testbench software from GitHub (https://github.com/hortonworks/hive-testbench.git)
+and use git to patch the beeline versions of the scripts into the hive-testbench directory.
+
+There are some instances where the metastore and database paths
+can become 'disjointed' Ex: the Metastore is configured for hdfs://node1/xxxx
+however the paths within the database report hdfs://node2/xxxx.  Sourcing the
+magpie-env.sh file and running the magpie-hive-update-table-paths.sh script should
+resolve this situation.
```

Comment:
perhaps add some text to inform the user that they should run ```tpch-build.sh``` before trying testbench.